cms/core/blocks/headline_figures.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any
+
+from django.utils.translation import gettext as _
+from wagtail.blocks import CharBlock, ListBlock, StructBlock
+
+
+class HeadlineFiguresItemBlock(StructBlock):
+    """Represents a headline figure."""
+
+    title = CharBlock(label=_("Title"), max_length=60, required=True)
+    figure = CharBlock(label=_("Figure"), max_length=10, required=True)
+    supporting_text = CharBlock(label=_("Supporting text"), max_length=100, required=True)
+
+
+class HeadlineFiguresBlock(ListBlock):
+    """A list of headline figures."""
+
+    def __init__(self, search_index: bool = True, **kwargs: Any) -> None:
+        kwargs.setdefault("max_num", 4)
```

Comment:
If this will be sharing the same design as in the prototype:

```suggestion
        kwargs.setdefault("max_num", 3)
```

Reply:
alas, this was specified as max 4.

Reply:
That's because the DS component will support up to 4.